src/wp-includes/interactivity-api/class-wp-interactivity-api.php

```diff
@@ -180,6 +211,28 @@ public function print_client_interactivity_data() {
 		}
 	}
 
+	/**
+	 * Returns the latest value on the context stack with the passed namespace.
+	 *
+	 * When the namespace is omitted, it uses the current namespace on the
+	 * namespace stack.
+	 *
+	 * @since 6.6.0
+	 *
+	 * @param string $store_namespace Optional. The unique store namespace identifier.
+	 */
+	public function get_context( string $store_namespace = '' ): array {
```

Comment:
It seems preferable to me to use `null` to indicate "doesn't exist" instead of empty string. We could also add "doing it wrong" messages here if empty string is passed.

```suggestion
	public function get_context( ?string $store_namespace = null ): array {
```

Reply:
Fixed in https://github.com/WordPress/wordpress-develop/pull/6394/commits/42e3e625dbb91cb82ff5cb702fe04c2369e39c33.